tests/data/one_element_subscript.py

```diff
@@ -0,0 +1,24 @@
+# We should not treat the trailing comma
```

Comment:
Can you try some cases like `[int,]` in a list, outside a subscript?

Reply:
have added the extra test cases and all looks good 😄